plugins/index.md

```diff
@@ -136,15 +141,83 @@ Plugins are tools to extend and modify the functionality of Foreman. The core of
+
+## 2.1 Using the Foreman installer
+
+Some plugins can be installed using the Foreman installer, which can help with additional configuration that the plugin may require, e.g. config files or services.
+
+To see which plugins the installer can set up, run:
+
+    # foreman-installer --help | grep enable-
+        --[no-]enable-foreman-plugin-discovery Enable 'foreman_plugin_discovery' puppet module (default: false)
+        --[no-]enable-foreman-plugin-docker Enable 'foreman_plugin_docker' puppet module (default: false)
+        --[no-]enable-foreman-plugin-hooks Enable 'foreman_plugin_hooks' puppet module (default: false)
+
+Run the installer with the extra `--enable...` argument to add the plugin.  Any arguments given when Foreman was first installed will be kept the same, saved in the installer answers file.
+
+Note that the installer will enforce the state of all managed configuration files, so manual changes will be reverted.  Use `--noop -v` first to check for any unexpected changes.
+
+    foreman-installer --noop -v --enable-foreman-plugin-discovery
+
+Run again without noop to make the changes:
+
+    foreman-installer --enable-foreman-plugin-discovery
+
+Some plugins have additional parameters, which can be changed using `foreman-installer -i` for interactive mode or found in the `foreman-installer --help` output.
 
-## 2.1 RPM installations
-The repositories are available at [yum.theforeman.org/plugins](http://yum.theforeman.org/plugins/).  Separate repositories are available for each Foreman release, containing plugins that are compatible with that particular version.  Packages are not currently GPG signed.
+## 2.2 Package installation
 
-* Configure the repo by creating `/etc/yum.repos.d/foreman_plugins.repo` with the following content:
+#### Select operating system
+
+<script type="text/javascript">
+function update_installation_os(select) {
+  var os = select.value;
+  $(".installation_os").hide();
+  if (os && os != 'none') {
+    $(".installation_os_"+os).show();
+  } else {
+    $(".installation_os_none").show();
+  }
+}
+</script>
+
+To provide specific installation instructions, please select your operating system:
+<select onChange="update_installation_os(this);">
+  <option value="none">-- select operating system --</option>
+  <option value="debian">Debian or Ubuntu</option>
+  <option value="el6">Enterprise Linux 6</option>
+  <option value="el7">Enterprise Linux 7</option>
+  <option value="fedora19">Fedora 19</option>
+</select>
+
+<div class="installation_os installation_os_none">
+  <i>No operating system selected.</i>
+</div>
+
+<div class="installation_os installation_os_el6 installation_os_el7 installation_os_fedora19">
+  <p>
+    The repositories are available at [yum.theforeman.org/plugins](http://yum.theforeman.org/plugins/).  Separate repositories are available for each Foreman release, containing plugins that are compatible with that particular version.  Packages are not currently GPG signed.
```

Comment:
The link to yum.theforeman.org.plugins is not working, probably because it's markdown syntax inside html. An `<a href>` tag works.